### ISSUE TITLE
FAQ

### ISSUE THREAD

**Issue by ThomasLeigh** [yanus171/Handy-News-Reader#583]
> Read the FAQ below or on [our website](https://handynewsreader.blogspot.com/2020/07/frequently-asked-questions.html).

<blockquote>
<p><sup><strong>Updated:</strong> July 8, 2020</sup></p>
</blockquote>
<p><strong>Quick survey on what’s coming:</strong></p>
<ol>
<li>What are the differences between HNR published on Google Play and F-Droid?</li>
<li>What is the approach about ads?</li>
<li>I want to support Your app financially speaking - how can I do that?</li>
<li>How to ensure that particular articles never will be deleted?</li>
<li>Will “Delete read articles” work if I have “Articles preservation time” set to “Forever”?</li>
<li>What for and when exactly “Article reading notification” is displayed?</li>
<li>Is it possible to load images only within the article I want at the moment, i.e., on d6emand - while having “Display images” disabled in the app’s settings?</li>
<li>Why there are no various-colored squares with letters alongside articles on article list?</li>
<li>Are non-regex filters case-sensitive?</li>
<li>Filters (a feed edit screen): how multiple keywords should be separated?</li>
<li>Where does “Auto set the article as read once its title has been visible on the list” should work?</li>
<li>How does “Reset feed and delete non-starred articles” work?</li>
<li>How does “Web Page - Divide” load type work?</li>
<li>What’s the difference between “Web Page - Links” and “Web Page - Divide” load types?</li>
<li>How the app knows how many images and texts are out there to be loaded - before I start the refresh?</li>
</ol>
<h2 id="what-are-the-differences-between-hnr-published-on-google-play-vs-f-droid-vs-github">What are the differences between HNR published on Google Play vs F-Droid vs GitHub?</h2>
<p><strong>Google Play Store</strong> is the place where official <u>stable</u> version of the app shows up.</p>
<p><strong>GitHub</strong> You might consider a place dedicated for both stable and beta versions of the app - therefore new versions lands there much more often.</p>
<p><strong>F-Droid</strong> obtains HNR version from GitHub, but with a significant delay (even up to a few weeks).</p>
<h2 id="what-is-the-approach-about-ads">What is the approach about ads?</h2>
<p>Here is the developer’s approach:</p>
<blockquote>
<p>“I can commit that the app would never include ads. This is my point of view, especially because it’s one of the main features: read content without useless content - ads.</p>
</blockquote>
<blockquote>
<p>On the other hand the app is open source, so in case the ads would appear in the app, any person with near-developer skills can cut the ads from the app and compile and release an ad-free app version.”</p>
</blockquote>
<h2 id="i-want-to-support-your-app-financially-speaking-how-can-i-do-that">I want to support Your app financially speaking - how can I do that?</h2>
<p><a href="https://money.yandex.ru/to/41001574414913" title="Donate the app development.">Here</a> You can do that.</p>
<h2 id="how-to-ensure-that-particular-articles-never-will-be-deleted">How to ensure that particular articles never will be deleted?</h2>
<p>The only way to do that is to add them to Favorites/to star them.</p>
<h2 id="will-delete-read-articles-work-if-i-have-articles-preservation-time-set-to-forever">Will “Delete read articles” work if I have “Articles preservation time” set to “Forever”?</h2>
<p>Yes, it has a higher priority over articles preservation.</p>
<h2 id="what-for-and-when-exactly-article-reading-notification-is-displayed">What for and when exactly “Article reading notification” is displayed?</h2>
<p>When You’re reading an article and then switch to other app for a while, Android may decide to shut down Handy to free up memory. As a result when You return to Handy, it may need to launch everything from scratch (which in case of longer articles may consume a noticeable amount of time).</p>
<p>Here “Article reading notification” comes in handy: since it’s a persistent notification, it lets Android know that You want Handy to be up-and-running, therefore it should not be automatically closed. As a result switching back to Handy should always be lightning-quick :) .</p>
<h2 id="is-it-possible-to-load-images-only-within-the-article-i-want-at-the-moment-i.e.on-demand-while-having-display-images-disabled-in-the-apps-settings">Is it possible to load images only within the article I want at the moment, i.e., on demand - while having “Display images” disabled in the app’s settings?</h2>
<p>Yes, just tap <em>Reload › Load all images</em> from the article context menu.</p>
<h2 id="why-there-are-no-various-colored-squares-with-letters-alongside-articles-on-article-list">Why there are no various-colored squares with letters alongside articles on article list?</h2>
<p>At some point of the app’s development they have been removed to make more space for article titles.</p>
<h2 id="are-non-regex-filters-case-sensitive">Are non-regex filters case-sensitive?</h2>
<p>No.</p>
<h2 id="filters-a-feed-edit-screen-how-multiple-keywords-should-be-separated">Filters (a feed edit screen): how multiple keywords should be separated?</h2>
<p>A feed-related filter can operate only on a single keyword.</p>
<h2 id="where-does-auto-set-the-article-as-read-once-its-title-has-been-visible-on-the-list-should-work">Where does “Auto set the article as read once its title has been visible on the list” should work?</h2>
<p>Only within a single feed list (therefore it’s not applied to the “Unread”, “Starred”, and other lists).</p>
<h2 id="how-does-reset-feed-and-delete-non-starred-articles-work">How does “Reset feed and delete non-starred articles” work?</h2>
<p>This option makes the feed as if it has just been added, so when a refresh is made, a new clean download of its articles will be made.</p>
<h2 id="how-does-web-page-divide-load-type-work">How does “Web Page - Divide” load type work?</h2>
<p>It has been created as a feature suited for forums (discussion boards) with no RSS. Thanks to this load type You can load posts as separate articles. To set up such a mechanism You need to point out ‘what is what’ (what is <em>a post</em> = <em>an article</em>, what is its <em>date</em>, <em>avatar</em>, etc.) - based on HTML classes. To determine class names, go to <em>Edit Content View</em>, find the proper class, tap on it and choose <em>Copy class name</em>.</p>
<h2 id="whats-the-difference-between-web-page-links-and-web-page-divide-load-types">What’s the difference between “Web Page - Links” and “Web Page - Divide” load types?</h2>
<p><strong>“Web Page - Divide”</strong> is meant for pages which full content is already there (on a page), organized into logical separate blocks, like single posts within a forum thread.</p>
<p><strong>“Web Page - Links”</strong> simply follows and loads all the links present on a given page (You can further adjust which links in particular should be loaded and which omitted).</p>
<h2 id="how-does-the-app-know-how-many-images-and-texts-are-out-there-to-be-loaded-before-i-start-the-refresh">How does the app know how many images and texts are out there to be loaded - before I start the refresh?</h2>
<p>It knows this from the previous refresh, when Handy created a task list consisting of all the images and texts which should be downloaded. Oftentimes the whole task list cannot be realized at once, due to:</p>
<ol>
<li>Android saving battery power, therefore not allowing Handy to do its job 100%.</li>
</ol>
<p>or</p>
<ol start="2">
<li>Weak Internet connection or server-side issues.</li>
</ol>
<p>In such cases Handy will take another - up to three - attempts to download all the missing images and texts during the next refresh. In case the issues persist, the app will remove missing items from the task list not to bother anymore.</p>
<p>Handy may display the overall remaining images (prefixed with “I”) and texts (prefixed with “T”) counts on the sidebar (under “All Articles” item).</p>

**Comment by yanus171** [comment]
Can we add link to FAQ in the app about sections?

**Comment by ThomasLeigh** [comment]
Done.

**Comment by yanus171** [comment]
Did you add it to app about section too?

**Comment by ThomasLeigh** [comment]
I added it to ReadMe.md on this GitHub. On HNR website there is a separate menu item for “FAQ”. What do I miss?

**Comment by yanus171** [comment]
There is about section in the app

**Comment by ThomasLeigh** [comment]
Oh, could You point out where can I do that?

**Comment by yanus171** [comment]
about_us_content in 
https://github.com/yanus171/Handy-News-Reader/blob/master/FlymFork/src/main/res/values/strings.xml

**Comment by ThomasLeigh** [comment]
done

**Comment by nikhilCad** [comment]
About the point no. 8. Are those disabled even for feeds having an icon? Because none of my feeds show any icon beside their title. 

And if that is not the case, can give some example of feed for which the icons work?

**Comment by nikhilCad** [comment]
Yes the "Show article icon / Turn off to gain more space of screen" is
checked. Can you give a feed for which icons show cause I want to see if
its problems with my feeds or something else

**Comment by yanus171** [comment]
http://android-developers.googleblog.com/feeds/posts/default

**Comment by nikhilCad** [comment]
doesn't show
![Screenshot_20220426-073853](https://user-images.githubusercontent.com/47214394/165205318-be9bf242-ef71-4abf-8e46-1acd47ba3834.png)

**Comment by yanus171** [comment]
Are the images shown in the article texts?

**Comment by nikhilCad** [comment]
Yes images are shown

**Comment by yanus171** [comment]
Mail me your HNR backup file. I am interested in settings (prefs) section, 
so you can cut all other info from it using text editor
